### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by Mirror-Tang**
Fix broken link: Learn more about MACI

## Description
The original link is invalid and has been updated to the correct link.

https://ethereum.org/zh/zero-knowledge-proofs/#how-maci-works-with-zk-proofs

### CODE REVIEW

**Review by minimalsm — APPROVED** [review]
Thanks @Mirror-Tang ❤️

File: src/content/zero-knowledge-proofs/index.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ Using MACI _does_ require trusting the coordinator not to collude with bribers o
 
 But in cases where the coordinator remains honest, MACI represents a powerful tool for guaranteeing the sanctity of on-chain voting. This explains its popularity among quadratic funding applications (e.g., [clr.fund](https://clr.fund/#/about/maci)) that rely heavily on the integrity of each individual's voting choices.
 
-[Learn more about MACI](https://github.com/privacy-scaling-explorations/maci/blob/master/specs/01_introduction.md).
+[Learn more about MACI](https://privacy-scaling-explorations.github.io/maci/index.html).
```
Comment (nhsz) [inline src/content/zero-knowledge-proofs/index.md:179]: Thanks for the fix! Suggesting to use just `https://privacy-scaling-explorations.github.io/maci/` for the link

**Review by nhsz — CHANGES_REQUESTED** [review]

File: src/content/zero-knowledge-proofs/index.md (hunk 1 above)
Comment (Mirror-Tang) [reply src/content/zero-knowledge-proofs/index.md:179]: https://github.com/ethereum/ethereum-org-website/pull/11101

Comment (Mirror-Tang) [reply src/content/zero-knowledge-proofs/index.md:179]: Modified in new pull request, please check for updates.
https://github.com/ethereum/ethereum-org-website/pull/11101

**Review by nhsz — APPROVED** [review]
lgtm

**Pull request merged**